scriptoocloud/src/main/java/edu/asu/diging/scriptoocloud/core/service/impl/TarWriter.java

```diff
@@ -0,0 +1,54 @@
+package edu.asu.diging.scriptoocloud.core.service.impl;
+
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.nio.file.FileSystems;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+import org.apache.commons.compress.archivers.tar.TarArchiveEntry;
+import org.apache.commons.compress.archivers.tar.TarArchiveOutputStream;
+
+public class TarWriter {
+
+    private FileOutputStream out;
+    private TarArchiveOutputStream tOut;    
+    private int len;
+    
+    public TarWriter(String outPath) throws IOException{        
+        out = new FileOutputStream(outPath + ".tar");        
```

Comment:
is this variable used anywhere else? If not, it doesn't need to be a class variable.